app/api/v1/endpoints/integrity.py

```
@@ -0,0 +1,314 @@
+        raise HTTPException(status_code=404, detail="Repair not found")
+    return _to_chain_repair_response(record)
+
+
+@router.post(
+    "/integrity/repair/{repair_id}/complete",
+    response_model=ChainRepairResponse,
+)
+async def complete_chain_repair(
+    repair_id: str,
+    tenant_id: Annotated[str, Depends(get_tenant_id)],
+    chain_repair_svc=Depends(get_chain_repair_service),
```

Ruff (0.15.5)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)